webapp/components/test-search.js

```diff
@@ -79,6 +79,7 @@ const QUERY_GRAMMAR = ohm.grammar(`
 
     Fragment
       = not Fragment -- not
+      | LinkExp
```

Comment:
nit: can we make this more consistent?
```suggestion
      | linkExp
```